src/components/LoggedOut/index.js

```diff
@@ -16,12 +16,18 @@ export default () => (
       <InstructorBanner className='w-100 mw6' />
     </div>
 
-    <div className='mb5'>
-      <Anchor url={getLoginUrl()}>
-        <Button>
+    <div className='mb5 tc center'>
+      <a 
+        href={getLoginUrl()}
+        className='no-underline'
+      >
```

Comment:
Using `<a>` for simple link without styles, `<Anchor>` for inline link with styles